server/game/cards/characters/06/bericdondarrion.js

```diff
@@ -0,0 +1,36 @@
+const DrawCard = require('../../../drawcard.js');
+
+class BericDondarrion extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: this,
+            effect: ability.effects.dynamicStrength(() => this.tokens['kiss'])
+        });
+
+        this.forcedReaction({
```

Comment:
Could you add a TODO comment here saying that we need a way for it not to be cancellable by things like Treachery? It'll be easy to forget otherwise. Or, alternatively, implement an ability flag that will prevent it from being cancellable.